_includes/components/header-and-footer/Modals/redux.js

```diff
@@ -10,7 +10,7 @@ export function createModal(title, markup) {
     type: CREATE_MODAL,
     payload: {
       title,
-      markup,
```

Comment:
revert please

Reply:
/Users/siwela/Documents/OpenUp/static-budget-portal/_includes/components/header-and-footer/Modals/redux.js

Reply:
Resolved, 'markup' back in the file: